apis/python/node/src/lib.rs

```diff
@@ -83,6 +83,10 @@ impl Node {
             self.send_output_slice(output_id, data.len(), data, metadata)
         })
     }
+
+    pub fn dataflow_descriptor(&self, py: Python) -> pythonize::Result<PyObject> {
```

Comment:
Nice, can we simply add a small method description as it is going to be shown in the API. thanks a lot!

Reply:
Yes, of course. Done in 482b282.